lib/datadog/tracing/contrib/dalli/quantize.rb

```diff
@@ -10,7 +10,21 @@ module Quantize
 
           def format_command(operation, args)
             placeholder = "#{operation} BLOB (OMITTED)"
-            command = [operation, *args].join(' ').strip
+            command = +operation.to_s
```

Comment:
[String#+@](https://ruby-doc.org/3.2.2/String.html#method-i-2B-40) is only supported in Ruby 2.3+.